csv/ParameterDefs.csv

```diff
@@ -763,7 +763,7 @@ DEPRECATED,DOSTA,estimated_oxygen_concentration,PD940,quantity,,float32,,µmol L
 ,DOSTA,dissolved_oxygen,PD953,quantity,,float32,,µmol kg-1,-9999999,Dissolved Oxygen Concentration,4,,,,,,,Calculated oxygen concentration,,,
 "CW_TEST,ADCPT_DE,ADCPS_IK,ADCPT_ACFGM_PD0_DCL,ADCPT_ACFGM_PD8_DCL",ADCP,upward_earth_seawater_velocity,PD960,function,"[""bin""]",float32,,m s-1,-9999999,Upward Seawater Velocity,4,,PFID128,"{""w"":""PD716""}",VELPROF-VLU,upward_sea_water_velocity,VELPROF-VLU_L1,A Velocity Profile includes water velocity (speed & direction) throughout the depth range of an ADCP sensor. This instance is the upward seawater velocity component. Units: m s-1,Upward Seawater Velocity Profile,Science Data,L1
 "CW_TEST,ADCPT_DE,ADCPS_IK,ADCPT_ACFGM_PD0_DCL,ADCPT_ACFGM_PD8_DCL",ADCP,earth_error_velocity,PD961,function,"[""bin""]",float32,,m s-1,-9999999,Error Velocity,4,,PFID129,"{""e"":""PD717""}",VELPROF-ERR,,VELPROF-ERR_L1,A Velocity Profile includes water velocity (speed & direction) throughout the depth range of an ADCP sensor. This instance is the error component. Units: m s-1,Error Seawater Velocity Profile,Science Data,L1
-"SP_TEST,CTD_CABLED","DOSTA, CTDBP",ctd_tc_oxygen,PD964,function,,float32,,umol L-1,-9999999,DO,4,,PFID60,"{""o2_counts"":""PD197""}",DOCONCS,,DOCONCS_L1,"Dissolved Oxygen (DO) Concentration from the Stable Response Dissolved Oxygen Instrument is a measure of the concentration of gaseous oxygen mixed in seawater. Units: umol L-1.",Dissolved Oxygen Concentration from Stable DO Instrument,Science Data,L1
+"SP_TEST,CTD_CABLED","DOSTA, CTDBP",ctd_tc_oxygen,PD964,function,,float32,,umol L-1,-9999999,DO,4,,PFID60,"{""o2_counts"":""PD197""}",DOCONCS,,DOCONCS_L1,Dissolved Oxygen (DO) Concentration from the Stable Response Dissolved Oxygen Instrument is a measure of the concentration of gaseous oxygen mixed in seawater. Units: umol L-1.,Dissolved Oxygen Concentration from Stable DO Instrument,Science Data,L1
```

Comment:
Since you're changing this, please do the following updates as well:
- umol -> µmol
- remove "Units: umol L-1." from the description field

Reply:
Is there a ticket associated with change umol to µmol ?  I can make the change, I just to accurately update any ticket with a reference to the change.

Reply:
No ticket. It's just that umol is not right. The correct prefix for micro is the Greek letter µ (mu), not the Latin letter u (ew). For the units, I spoke with the Data Team and they agreed that the units should not be in two places (possible divergence) so I've been fixing them as I make preload updates. Sometime in 2058, we'll have fixed them all.

Reply:
This particular line wasn't changed by me directly.  It was indirectly changed after the csv was written back out by the spreadsheet software I used.  Looks like it removed the quote because the field didn't contain an internal comma or quote character.

There's quite a few more umol's in ParameterDefs.csv, I think I'm going to write a ticket to have someone handle all of them at once.

Reply:
sounds like a plan